package.json

```diff
@@ -8,7 +8,7 @@
   "license": "ISC",
   "scripts": {
     "start": "webpack && node server",
-    "dev": "node server & webpack --watch",
+    "dev": "nodemon server & webpack --watch",
```

Comment:
Please remove this change as it is irrelevant to your ticket. If this change is needed create a new ticket please.